test/plugin.test.ts

```diff
@@ -18,6 +18,23 @@ function normalizeSlashes(path: string | null): string | null {
   return path === null ? null : path.replace(/\\/g, '/');
 }
 
+function testBothModes(
+  testFunction: (d: string, func: () => Promise<void>, options: TestOptions) => void,
+  description: string,
+  f: (mode: string, suffix: string) => Promise<void>,
+  testOptions: TestOptions,
+) {
+  testFunction ||= it
+  const MODES = [["production", MODE_PRODUCTION, "opt"], ["development", MODE_DEVELOPMENT, "fastopt"]]
+  MODES.forEach( ([modeName, mode, suffix]) => {
+    testFunction(
+      description + "(" + modeName + ")",
```

Comment:
Nit:
```suggestion
      description + " (" + modeName + ")",
```
will probably display better.

Reply:
Added.